superbench/config/azure_ndmv4.yaml

```diff
@@ -3,7 +3,7 @@
 # Azure NDm A100 v4
```

Comment:
Can we add more examples on h100 and mi200 skus?

Reply:
added example file for NDV5 and mi300x to include hipblast/cublaslt/fp8 training/dist-inference